test/typed_buffers_test.dart

```diff
@@ -485,8 +486,8 @@ void testInt32x4Buffer(List<int> intSamples) {
     expect(buffer.length, equals(0));
 
     var samples = intSamples
-        .where((value) => value == rounder(value))   // Issue 15130
-        .map((value) => new Int32x4(value, -value, ~value, ~-value))
+        .where((value) => value == rounder(value)) // Issue 15130
```

Comment:
Issue 15130 has been fixed, so I think you can just  delete this line.

Reply:
ack